lib/git_reflow/git_server/git_hub.rb

```diff
@@ -115,15 +115,20 @@ def authenticate(options = {silent: false})
 
           rescue ::Github::Error::Unauthorized => e
             if e.inspect.to_s.include?('two-factor')
-              two_factor_code = ask("Please enter your two-factor authentication code: ")
-              self.authenticate options.merge({user: gh_user, password: gh_password, two_factor_auth_code: two_factor_code})
+              begin
+                # dummy request to trigger a 2FA SMS since a HTTP GET won't do it
+                @connection.oauth.create scopes: ['repo'], note: "thank Github for not making this straightforward"
```

Comment:
Reference: https://github.com/jingweno/gh/issues/206.  Github won't send an SMS with the two-factor authentication code unless the call is a `POST` or `PUT`